e2e/shells/utils.ts

```diff
@@ -40,6 +40,7 @@ const clean = (chunk: string) =>
   debug(
     stripAnsi(chunk)
       .replaceAll(BEL, ``)
+      .replaceAll(/\r\n|\r|\n/, `\n`)
```

Comment:
`replaceAll` needs to be called with a global regexp. Also, are there any cases on Windows where only `\r` is present without `\n`? Usually when normalizing line endings in Yarn we use `/\r?\n/g`.

```suggestion
      .replaceAll(/\r\n|\r|\n/g, `\n`)
```

Reply:
I'm not sure to be honest, I wasn't sure if there were cases when dealing with a terminal where you might get \r without \n.